shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/network/helpers/TargetType.kt

```diff
@@ -0,0 +1,23 @@
+package co.nimblehq.blisskmmic.data.network.helpers
+
+import io.ktor.client.request.*
+import io.ktor.http.*
+
+interface TargetType {
+
+    var baseURL: String
+    var path: String
+    var method: HttpMethod
+    var body: Any?
+    var headers: Map<String, String>?
+
+    val requestBuilder: HttpRequestBuilder
+    get() {
+        val builder = HttpRequestBuilder()
+        builder.url("$baseURL$API_VERSION$path")
```

Comment:
I think we can force base url here

```suggestion
        builder.url("${BuildKonfig.BASE_URL}$API_VERSION$path")
```